internal/domain/power-status.go

```diff
@@ -107,7 +109,9 @@ var powerStatusMasterInterval = 15
 func PowerStatusMonitorInit(domGlb *DOMAIN_GLOBALS,
                             distLockMaxTimeIn time.Duration,
                             loggerIn *logrus.Logger,
-                            sampleInterval time.Duration) error {
+                            sampleInterval time.Duration,
+														bmcTimeout int,
+														bmcRetries int) error {
```

Comment:
Minor nit... maybe these should be called statusTimeout and statusRetries to match the names that were called in?  Feel free to ignore.